### DISCUSSION TITLE
State of Lutra

### DISCUSSION THREAD

**Discussion by JacobCoffee** [pradyunsg/lutra#45] [Questions] [answered]
This is by far the best looking documentation theme. I am salivating.

1. Is this project active? Is it going anywhere?
2. Are you looking for contributors?
3. What can I do to help make this production-ready?

**Comment by pradyunsg** [comment, marked as answer]
> Is this project active? Is it going anywhere?

Yes, and yes. This has stalled a bit due to lack of time on my end and a few more urgent pieces (eg: cutting a pip release, broader strategy discussions for Python packaging) taking up my limited free time instead.

> Are you looking for contributors?

Yes! I’d love to increase the bus factor on this project.

> What can I do to help make this production-ready?

Picking up the most well defined pieces? There’s a bunch of open-ended things like implementing proper footers, lots of documentation to write etc — but, there’s also things like https://github.com/pradyunsg/lutra/issues/36, https://github.com/pradyunsg/lutra/issues/9, which are easier to point at and say do this where it’s unlikely that we’d end up looking at the issue and come out with different opinions. :)
